### PR TITLE
feat:Added Todo

### PR DESCRIPTION
**Pull request by tushargoyal812**
https://user-images.githubusercontent.com/69681896/164933847-ea4e00bf-75fd-43ac-a8b8-578361fb2d38.mp4

### CODE REVIEW

File: chrome-extension/src/App.js [hunk 1]
```diff
@@ -18,13 +18,29 @@ function App() {
...
+  const [todoValues,setTodoValues]=useState([])
+  const [editId,setEditId]=useState()
+
+  const addHandler=(value)=>{
+    setTodoValues([...todoValues,{id:uuid(),todoName:value,isDone:false}])
+  setTodoInput("")
+}
+
+  const todoInputHandler=(id)=>{
+  let newValue=todoValues.map(item=>item.id===id?{...item,isDone:!item.isDone}:item)
```
Comment (deepak29-git) [inline chrome-extension/src/App.js]: try to use good variable name instead of `newValue` like newTodo

File: chrome-extension/src/App.js [hunk 2]
```diff
@@ -63,34 +93,53 @@ function App() {
...
       <h1 className='main-heading'>{`${greetings} ${userName}`}</h1>
       <h1 className='main-heading'>What's your main focus for today?</h1>
       <input className='detail-input' type="password" />
       <div>
-      <button onClick={passwordHandler} className='Countinue-btn'>Continue</button>
+      <button onClick={passwordHandler} className='Countinue-btn cursor-pointer'>Continue</button>
       </div>
+      {showModal&&<div className='ToDo-modal-container'>
+        <div className='ToDo-modal'>
+          {console.log(todoValues,"from below")}
```
Comment (deepak29-git) [inline chrome-extension/src/App.js]: remove console after debugging

**Review by deepak29-git — APPROVED** [review]

**Review by sharmayatendra — APPROVED** [review]
clean code
good file structure  
good to merge

File: chrome-extension/src/App.js (hunk 1 above)
Comment (tushargoyal812) [reply chrome-extension/src/App.js]: ok

File: chrome-extension/src/App.js (hunk 2 above)
Comment (tushargoyal812) [reply chrome-extension/src/App.js]: done

**Pull request merged**